index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nove Theme</title>
+    <!-- Favicon -->
+    <link rel="shortcut icon" href="./images/favicon.png" type="image/x-icon">
+    <!-- Styles Links -->
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/unit.css">
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./styles/fonts.css">
+
+</head>
+<body>
+<!-- Header Section -->
+    <header>
+        <section class="row header__section">
```

Comment:
Skoro mamy ELement z BEM tj. `header__section` to  w hierarchi HTML powinien wystąpić Block z BEM.
Czyli w tym przypadku znacznik header powinien wyglądać  tak:  `<header class="header">`

Więcej o BEM znajdziesz tutaj: https://devmentor.pl/b/metodologia-bem-w-css-i-sass